app/assets/v2/js/lib/vue.js

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
moved the medium images and vue.js images to our own site so they could be cached according to our CDN config..